### DISCUSSION TITLE
真實的抽象

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1350] [Ideas]
請你注意一件事，正如「上帝」不是「任何存在者」，所謂「貨幣」也不是「金銀」，它不是任何具體的存在，而是一種「抽象的介質」，但正是因為這個抽象介質，世界才會顯現為具象的、可以被理解的樣子。

馬克思寫道：

- 「一種商品成為貨幣，似乎不是因為其他商品都通過它來表現自己的價值，相反，似乎因為這種商品是貨幣，其他商品才能通它來表現自己的價值。中介運動在它本身的結果中消失了，而且沒有留下任何痕跡。」《資本論》第一卷 p112

這段話令人費解，卻至關重要，它隱晦地表達了一種「中介退隱，但秩序顯現」的邏輯，這是我們把握貨幣與商品關係的關鍵，但並不容易說清楚，我們要藉助類比思維來理解。

你應該注意到了，馬克思在面前給出了兩個非常古怪的說法：“商品的鏡子”和“商品的語言”。我會向你證明，用視覺系統和語言系統的運作來類比理解商品系統的運作，能幫我們開一個非常大腦洞。

先說視覺系統，語言系統放在下一節。

先看看「中介運動在它本身的結果中消失了，沒有留下任何痕跡」是什麼意思。

作為貨幣時代原住民，我們能很難想像一個沒有貨幣的世界，那是一個陰暗混沌的物的世界，萬物是混作一團的，貨幣像一道光，讓萬物在一個序列中獲得了秩序。光的中介之下，我們看見彼此，光為物質現象界提供了最基本的秩序，可見光本身卻變成“不可見”的東西隱去了。

在一個晴朗的大白天，我們不會意識到自己是先看見了光，才看見了對象。 “不可見的可見光”暗中規定了我們與萬物的基本關係，大自然才會顯得自然而然，類似的，正因為有背景般的“貨幣等價形式”存在，我們才能自然而然地理解商品的關係。

理解了這個說法，再回過頭看「商品的鏡子」。

我們確認一件上衣能換十匹麻布，十匹麻布此時並不是十匹麻布，而是一件上衣表現自身的價值形式（等價形式），或者用馬克思的說法，麻布是一件上衣「表現自身價值的鏡子」。注意，這個鏡子並不一定得是麻布，它可以是任何東西，只要它處於鏡子的位置之上，也就是處於為其他對象提供價值形式的位置之上，它就成為了貨幣的化身。

鏡子這個隱喻非常妙。鏡子脫離了光，什麼都不是，一旦被照見了光，它卻可以在自身什麼都不是的情況下讓萬物確認自身。

按照馬克思的說法，金銀成了所有商品的鏡子，所有商品也成了金銀的鏡像。我們既可以說金銀是其他商品的一般等價物，也可以說其他商品都是金銀的特殊等價物。金銀成了絕對商品，成為能中介其他一切商品的絕對相對物。

- 「一個商品處於一般等價形式，是因為而且只是因為它被其他一切商品當做等價物『排擠出來』。這種排擠的結果最終只剩下一種獨特的商品，從這個時候起，商品世界的統一的相對價值形式才獲得客觀的固定性和一般的社會效力……有一個特定的商品在歷史過程中奪得了這個特權地位，這就是金。」《資本論》第一卷p86

今天你我手中的鈔票，就是我們用來理解萬物的鏡子，我們審視一件事，無非是審視這件事的損益比，“打量”一個人，無非是看他處於什麼價位。如此，我們才被嵌入了一個真實且抽象「價值形式」之中，進入了一個由貨幣秩序統治的商品現象界。

請你務必理解「貨幣等價形式」之於你我的影響，這種影響隱秘而廣泛，常識背後的常識，是背景之外的背景。假設一種狀況，外星人突然降臨，抹去了人類所有關於貨幣-金錢的知識，眼前的世界會立即變成一個巨大的謎，曾經熟悉的人事物會變成無法理解的存在，陷入前所未有的混亂。 （相比生物性的失明，這是社會維度的致盲，讀者裡面如果有編劇，可以演繹一下這個故事，會非常有趣）

小結：馬克思對貝利的補充是，必須有貨幣的介入，商品的相對價值在才會在一個價值體系中成立。一斤小麥、一頭牛、一匹麻布一旦“照了鏡子”，便瞬間確定了與世界上其他所有商品的相對關係，在“商品的無限系列”中找到自己的位置，完成了自我確認，有了定價。這個確認-定價合不合理另說，一次認識論上的飛躍卻完成了，一個叫麻布的商品誕生了，在一個有光的世界——有貨幣價值的世界——麻布可以用來換一切東西，只是份額不同而已 。

這裡，我們可以歸納出一個「中介退隱，萬物確認自身，相對而立」的邏輯。確認了自身，也就是確認了與他物的相對性，這是「商業流通」的前提，也是利潤的根本來源。

**Comment by 710388** [comment]
有聽沒有懂 太複雜了

**Comment by NcuJerryKao** [comment]
你提出的觀點非常有趣，關於「貨幣」的本質和作用，我想進一步探討一下。

[首先，讓我們回顧一下「貨幣」的定義。在經濟學中，貨幣是一種公認的交換媒介，用於簡化交易並降低交易成本。它具有計價的功能，可以衡量不同商品之間的價值，並作為價值的保存工具](https://bing.com/search?q=%e9%87%91%e9%8c%a2%e7%9a%84%e6%84%8f%e7%be%a9)[1](https://bing.com/search?q=%E9%87%91%E9%8C%A2%E7%9A%84%E6%84%8F%E7%BE%A9)。

你提到了馬克思的觀點，他在《資本論》中探討了貨幣的角色。其中，「商品的鏡子」和「商品的語言」是他用來描述貨幣的隱喻。讓我們來深入理解一下：

[商品的鏡子：假設一件上衣能換十匹麻布，那麼這十匹麻布就不再只是麻布，而是一件上衣的價值形式。這種轉換使得麻布成為了一種「表現自身價值的鏡子」，它代表著其他商品的價值](https://money.cmoney.tw/article/13114)[2](https://money.cmoney.tw/article/13114)。
[商品的語言：貨幣作為一種普遍接受的交換媒介，讓不同商品之間的價值可以用共同的語言來表達。這種語言的存在使得商品之間的交換更加方便，並且有助於確定價值](https://money.cmoney.tw/article/13114)[2](https://money.cmoney.tw/article/13114)。
總之，貨幣不僅是一種物理存在，更是一種抽象的介質，它在交易中扮演著中介的角色，讓世界以具象的、可以被理解的樣子呈現。這種中介退隱的邏輯，使得商品的價值得以確認，商業流通和利潤也因此成為可能。

謝謝你的觀點，這讓我們更深入地思考了貨幣的本質和功能。如果你還有其他問題或想進一步討論，歡迎隨時提出！🙂

**Reply by yubo0oo** [reply]
@NcuJerryKao 
@benctw 
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。
